### PR TITLE
chore: add plop generator for new packages

### PR DESCRIPTION
**Pull request by nylon22**

### CODE REVIEW

File: plop-templates/package/package.json.hbs [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  "description": "{{packageDescription}}",
+  "main": "index.js",
+  "keywords": [{{{packageKeywords}}}],
+  "author": "{{userFullName}} <{{userEmail}}>",
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
+  "devDependencies": {
+    "react-dom": "^16.8.6",
```
Comment (GoPro16) [inline plop-templates/package/package.json.hbs:17]: Still need "react" as devDep here.

Comment (nylon22) [reply plop-templates/package/package.json.hbs:17]: 5bd9e33c3fe48107b59674050d622ec865b7b79c

Comment (TheSharpieOne) [reply plop-templates/package/package.json.hbs:17]: What about `prop-types` ?

Comment (nylon22) [reply plop-templates/package/package.json.hbs:17]: e0bd8371d92c475b874e732e24050b87153628e6

**Pull request merged**